image/cli/mascli/functions/install

```diff
@@ -386,6 +388,9 @@ function install_noninteractive() {
     --uds-lastname)
     export UDS_CONTACT_LASTNAME=$1 && shift
     ;;
+    --accept-license)
+    export LICENSE_ACCEPTED=true && shift
```

Comment:
If this is just a boolean flag i.e. `--accept-license` with no value then we don't need the shift at the end. This should probably be under the "Other commands" section below https://github.com/ibm-mas/cli/pull/437/files#diff-bd207331dfe68009e28fb66d98bb4719786c180b4716ed61ab76d82105179f7dR471